pkg/mover/chart.go

```diff
@@ -265,8 +309,12 @@ func (cm *ChartMover) printMove() {
 		if change.ShouldPush() {
 			pushRequiredTxt = "push required"
 		}
+		src := change.ImageReference.Name()
+		if cm.bundle != nil {
```

Comment:
in which case are we supposed to have bundle nil?

Reply:
when there is no bundle?

Move had 2 behaviors, regular and from bundle. This code is just here so we show that the origin of the images is the bundle, even if we also show the original tags of the images at origin.
For regular move the output we had remains the same.